src/libs/Localize/index.ts

```diff
@@ -197,7 +225,7 @@ function translateIfPhraseKey(message: MaybePhraseKey | ReceiptError): string |
             return phrase;
         }
 
-        return translateLocal(phrase as TranslationPaths, variables as never);
```

Comment:
Is there a reason to remove `variables` here?

Reply:
Found the reason! It's because `never[]` was changed to `[]` in `PhraseParameters` type.

``ts
type PhraseParameters<T> = T extends (...args: infer A) => string ? A : never[];
```